Makefile

```diff
@@ -13,8 +13,8 @@ EDEPINCDIR = $(EDEPSIM)/include/EDepSim
 
 all: Digitize Reconstruct Analyze
 
-struct.cxx: include/struct.h include/Linkdef.h
-	cd include && rootcint -f ../src/$@ -c $(CFLAGS) -p $(HEADERS) Linkdef.h && cd ..
+struct.cxx: include/struct.h include/StructLinkdef.h
```

Comment:
Usually, it doesn't make sense to have both a Makefile and a CMakeLists.txt in the same folder. Why is this Makefile here, and why is it being maintained?